chemie/events/models.py

```diff
@@ -22,6 +23,23 @@
     ("TRUANT", 3, "Ikke møtt"),
 )
 
+class BaseRegistrationGroup(models.Model):
+    name = models.CharField(max_length=100, verbose_name="Gruppenavn")
+    members = models.ManyToManyField(User, blank=True, verbose_name="Medlemmer")
+    def __str__(self):
+        return self.name
+
+    def add_member(self, user):
+        self.members+=user
```

Comment:
I assume you have tested this so it works, but I think the Django way to add to a ManyToManyField is with
`self.members.add(user)` as shown [here](https://docs.djangoproject.com/en/dev/ref/models/relations/)

Reply:
Fixed!